### ISSUE TITLE
Change pin for zero-cross in Arduino Mega ATMega2560

### ISSUE THREAD

**Issue by nicolasaw** [RobotDynOfficial/RBDDimmer#49]
Hi, is it possible to change the pin for zero-cross. I am currently using the pin D2 and i looked up and saw that for Mega, there are several other pin interrupts. I am planning to use pin D20, is there a way to configure it and how?

Thanks

**Comment by WackoKacko** [comment]
Yeah, this should be a standard feature. I may have a swing at it myself later. I'll make a pull request if I succeed.
